components/Buttons/src/ColorThemer/MDCContainedButtonColorThemer.h

```diff
@@ -27,7 +27,8 @@
 @interface MDCContainedButtonColorThemer : NSObject
 @end
 
-@interface MDCContainedButtonColorThemer (ToBeDeprecated)
+__deprecated_msg("Please use [MDCButton applyContainedThemeWithScheme:] instead. (Note: Color theming is no longer available as an independent API.)")
+    @interface MDCContainedButtonColorThemer (ToBeDeprecated)
```

Comment:
Deprecating the category doesn't cause warnings to be generated by usages of `MDCContainedButtonColorThemer`. This PR should remove the `@interface MDCContainedButtonColorThemer (ToBeDeprecated)` category declaration and apply the deprecation warning directly to the @interface declaration above on line 25.

Reply:
Done